src/seedu/addressbook/parser/Parser.java

```diff
@@ -75,33 +65,36 @@ public Command parseCommand(String userInput) {
 
         switch (commandWord) {
 
-        case AddCommand.COMMAND_WORD:
-            return prepareAdd(arguments);
+            case AddCommand.COMMAND_WORD:
+                return prepareAdd(arguments);
 
-        case DeleteCommand.COMMAND_WORD:
-            return prepareDelete(arguments);
+            case DeleteCommand.COMMAND_WORD:
```

Comment:
The switch case doesn't require indentation as per the coding style: https://oss-generic.github.io/process/codingStandards/CodingStandard-Java.html